src/paramTypes.ts

```diff
@@ -1,11 +1,13 @@
+import { ExchangeType } from 'menashmq';
```

Comment:
I'd add comments about the types explaining their meaning. Maybe even from the readme

Reply:
done, added a link to exchange types information in the readme file